src/main/groovy/org/edx/jenkins/dsl/JenkinsPublicConstants.groovy

```diff
@@ -15,6 +15,8 @@ class JenkinsPublicConstants {
     }
   }
 
+  public static final String JENKINS_PUBLIC_GITHUB_BASEURL = "http://github.com/"
```

Comment:
It should be https because that is what github uses and what is present in other non scripted pr jobs